### DISCUSSION TITLE
Why does the topology in version 9.5.0 look so strange?

### DISCUSSION THREAD

**Discussion by hzhswyz** [apache/skywalking#11132] [General]
I used version 9.3.0 before, and the topology looks quite normal. Are there any policies adjustments?
The current topology is very deep, it looks abnormal.

version 9.5.0 General-Root topology
![image](https://github.com/apache/skywalking/assets/44084348/817286b8-10ae-42a0-8570-aec83ccb7658)

**Comment by wu-sheng** [comment]
This means your dependencies are flat, not very distributed.
All new polices have been listed in 9.5 changelogs.
